packages/coinstac-ui/app/render/components/dashboard.jsx

```diff
@@ -36,7 +36,7 @@ class Dashboard extends Component {
       <div className="dashboard container-fluid">
         <div className="row">
           <div className="col-xs-12 col-sm-4">
-            <nav className="navigation" role="navigation">
+            <nav className="navigation">
```

Comment:
They are. The issue is including a role of `navigation` on a `nav` element: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-redundant-roles.md